dsn_test.go

```diff
@@ -608,6 +609,20 @@ func TestParseDSN(t *testing.T) {
 			ocspMode: ocspModeFailOpen,
 			err:      nil,
 		},
+		{
+			dsn: "u:p@a.r.c.snowflakecomputing.com/db/s?account=a.r.c&includeRetryReason=true",
```

Comment:
let's add a test with explicit false for the new parameter

Reply:
Done.